src/assets/styles/other/_material-reduction.scss

```diff
@@ -803,6 +803,12 @@ mat-card.mat-card .mat-list-base {
   margin-left: 8px;
 }
 
+@media screen and (max-width: 768px) {
+  .entity-table-controls > div {
+    margin-top: 8px !important;
```

Comment:
Can you do the same without important?

Reply:
The important here is needed for the search input. Other views take the margin without important but not the search input.

Reply:
If you are having a problem with `#row-filter` id, I'd prefer to have another css declaration for that rather than introducing `!important`.